test/mutation/mutation_pattern/test_position_pattern.py

```diff
@@ -20,19 +20,17 @@ def test_decode_position_pattern():
     test_stru = sp.get_structure(test_pdb)
     test_pattern = "resi 254 around 4"
 
-    assert set(m_p.decode_position_pattern(test_stru, test_pattern)) == set([
```

Comment:
yapf is actually making them looks worse lol. Can we prevent this?

Reply:
not sure tbh. I won't touch it for this commit but we can play around with the settings.